cli/cli/kurtosis_gateway/server/api_container_gateway/api_container_gateway_service_server.go

```diff
@@ -322,9 +326,9 @@ func (service *ApiContainerGatewayServiceServer) writeOverServiceInfoFieldsWithL
 	return nil
 }
 
-// startRunningConnectionForKurtosisService starts a port forwarding process from kernel assigned local ports to the remote service ports specified
+// startMaybeRunningConnectionForKurtosisService starts a port forwarding process from kernel assigned local ports to the remote service ports specified
 // If privatePortsFromApi is empty, an error is thrown
-func (service *ApiContainerGatewayServiceServer) startRunningConnectionForKurtosisService(serviceUuid string, privatePortsFromApi map[string]*kurtosis_core_rpc_api_bindings.Port) (*runningLocalServiceConnection, error) {
+func (service *ApiContainerGatewayServiceServer) startMaybeRunningConnectionForKurtosisService(serviceUuid string, privatePortsFromApi map[string]*kurtosis_core_rpc_api_bindings.Port) (*runningLocalServiceConnection, error) {
```

Comment:
Though my comment above doesn't really apply here. For those methods with long names, the Maybe is a bit unclear, we don't really know to which part of the name it applies. Maybe something like `startRunningConnectionForKurtosisServiceIfRunning` or `*Present`?

Reply:
I was trying to be as consistent as possible and I found function names starting with `getMaybe`.  I renamed the updated functions using a suffix and `IfRunning`.